composer.json

```diff
@@ -6,7 +6,7 @@
     "require": {
         "php": ">=7.4",
         "oliverde8/associative-array-simplified": "^1.0",
-        "psr/log": "^1.0",
+        "psr/log": "^2.0.0",
```

Comment:
Rather ^1.0|^2.0

for now as we don't want to drop support for symfony 5.4

Reply:
It also causes tests to fail at the moment as tests are still running on php 7.4.